public/css/wyr-single.css

```diff
@@ -42,4 +42,22 @@
   100% {
     transform: translateX(-150%);
   }
-}
+} */
+
+    #add-question-modal {
+      background-color: rgba(0, 0, 0, 0.5);
+    }
+
+    #add-question-modal.show {
+      animation: fadeIn 0.3s;
+    }
+
+    @keyframes fadeIn {
+      from {
+        opacity: 0;
+      }
+
+      to {
+        opacity: 1;
+      }
+    }
```

Comment:
above is extra styling to give faded background color for the #add-comment-modal and a slight fadeIn for the modal just like the adding comment modal